flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/StatementSetImpl.java

```diff
@@ -142,21 +143,17 @@ public TableResult execute() {
     }
 
     /**
-     * Get the json plan of the all statements and Tables as a batch.
-     *
-     * <p>The json plan is the string json representation of an optimized ExecNode plan for the
-     * statements and Tables. An ExecNode plan can be serialized to json plan, and a json plan can
-     * be deserialized to an ExecNode plan.
+     * Get the {@link CompiledPlan} of the all statements and Tables as a batch.
```

Comment:
Remove the JavaDocs of an `Impl` method. The interface should have all docs.

Reply:
This method is still not exposed. I'm tackling it in a separate pr here https://github.com/apache/flink/pull/18649